decidim-api/lib/decidim/api/test/component_context.rb

```diff
@@ -54,15 +54,21 @@
   let(:process_space_factory) { :participatory_process }
   let(:space_type) { "participatoryProcess" }
 
-  shared_examples "graphQL visible resource" do
-    it "is visible" do
-      expect(response[space_type]["components"].first[lookout_key]).to eq(query_result)
+  shared_examples "graphQL visible resource" do |visible: true|
+    if visible
+      it "is visible" do
```

Comment:
Please do check out my last comment: https://github.com/decidim/decidim/pull/15682/files/8a21a57354fd6728eee830d1a3f3d25d0a870e6f#r2609832379 

> Also the "is visible" is not consistent with the "it should not be visible" from lines 48 and 68.

```suggestion
      it "should be visible" do
```